app/utils/use-axis-title-adjustments.ts

```diff
@@ -33,12 +33,14 @@ export const useAxisTitleAdjustments = ({
   const overlapAmount =
     (axisTitleWidthLeft + axisTitleWidthRight) / containerWidth;
 
+  const axisLabelHeight = getTextHeight(leftAxisTitle, { fontSize });
```

Comment:
If I am not mistaken, here we have a "regular chart", with one title? I think if you passed `width: chartWidth` here, we could remove the `isOverlapping`, `overlapAmount` logic down below 🤔

Reply:
@bprusinowski This is for comboCharts, should I adjust the name to `use-dual-axis-title-adjustments` ?